component/rbac.libsonnet

```diff
@@ -0,0 +1,64 @@
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local inv = kap.inventory();
+local params = inv.parameters.control_api;
```

Comment:
We don't need params at all, as far as I can see, otherwise it should probably be `inv.parameters.openshift4_monitoring`.

```suggestion
```

Reply:
Ups yes. Copy pasta error..